zerodark-theme.el

```diff
@@ -30,6 +30,8 @@
 (require 's)
 (require 'all-the-icons)
 (require 'powerline)
+(when (require 'powerline-evil nil 'noerror)
+  (require 'powerline-evil))
```

Comment:
Whoops that's my bad I was trying something else before I went with this route it should just be:

`(require 'powerline-evil nil 'noerror)`